docs/mine-hnt/validators/validator-deployment-guide.mdx

```diff
@@ -564,5 +570,5 @@ Getting back on chain is easy now that you've got everything installed:
 
 - Follow the same process from [Stake Tokens to Your Validator](/mine-hnt/validators/validator-deployment-guide#stake-tokens-to-your-validator) to stake the 10,000 TNT received from the faucet for your Testnet validator.  If you kept the `swarm_key` file above, your validator node address will be the same.  If you deleted that file, you'll have to ensure you have the new node address by running `miner peer addr` as outlined in that section.
 
-That's it!  You're back on the new chain and should see similar output as defined in [Verifying Validator Victory](/mine-hnt/validators/validator-deployment-guide#verifying-validator-victory).
+Note, a validator node can only be staked once. That's it!  You're back on the new chain and should see similar output as defined in [Verifying Validator Victory](/mine-hnt/validators/validator-deployment-guide#verifying-validator-victory).
```

Comment:
This "Note, a validator node can only be staked once" feels out of place. Is there a better place for it?

Reply:
@jdgemm Just waiting oh this to be moved / tweaked and we can merge.